<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOptionsDockWidget</class>
 <widget class="QDockWidget" name="DisplayOptionsDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Display Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_Interpolation_2">
      <property name="title">
       <string>Drawing</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBoxRigidBodyMeshmodels">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Meshmodels</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_DrawFiltered">
         <property name="text">
          <string>Use filtered transformations</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBoxRigidBodyConstellation">
         <property name="text">
          <string>Rigid body constellations</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxMarkerIds">
         <property name="text">
          <string>Marker Ids</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxMarkers">
         <property name="text">
          <string>Marker cross</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBoxEpipolar">
         <property name="text">
          <string>Epipolar line</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxHide">
         <property name="text">
          <string>Hide all</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
